modules/ROOT/pages/kubernetes/quickstart-standalone/server-setup.adoc

```diff
@@ -27,5 +27,4 @@ Then, the Neo4j Helm chart creates Kubernetes entities needed for running and ac
 The following diagram is a schematic representation of the Helm chart involved and the Kubernetes and Cloud resources it instantiates when installed:
 
 .Neo4j standalone setup
-image::standalone-on-k8s.svg[Example of Neo4j standalone setup in Kubernetes,width=700,role=popup]
-
+image::standalone-on-k8s.svg[Example of Neo4j standalone setup in Kubernetes,width=700,role=popup]
```

Comment:
I don't think this finds the image
<img width="1596" height="348" alt="image" src="https://github.com/user-attachments/assets/06df25f2-ede4-4990-91e4-3a2ba76fd3df" />